lib/intacct/base.rb

```diff
@@ -170,8 +170,12 @@ def self.read_only_field(name)
 
     def self.formatted_error_message(errors)
       [errors].flatten.map do |error|
-        error["description"].presence || error["description2"] || "Undefined error"
-      end .join(" ")
+        if error["description"].present? && error["description2"].present?
+          "#{error["description"]}: #{error["description2"]}"
+        else
+          error["description"].presence || error["description2"] || "Undefined error"
```

Comment:
Should we also use `presence` with description2?
```suggestion
          error["description"].presence || error["description2"].presence || "Undefined error"
```